spyder/plugins/plots/widgets/figurebrowser.py

```diff
@@ -242,6 +249,23 @@ def setup_options_button(self):
         else:
             self.tools_layout.addWidget(self.options_button)
 
+    def create_shortcuts(self, parent):
+        """Create shortcuts for this widget"""
+        # Configurable
+        copyfig = config_shortcut(self.copy_figure, context='plots',
+                                  name='copy', parent=parent)
+
+        return [copyfig]
+
+    def get_shortcut_data(self):
+        """
+        Returns shortcut data, a list of tuples (shortcut, text, default)
+        shortcut (QShortcut or QAction instance)
```

Comment:
One blank line between summary line and main docstring body; end sentences with a period per PEP 257.